### DISCUSSION TITLE
need a small help

### DISCUSSION THREAD

**Discussion by sec13b** [projectdiscovery/httpx#1606] [Q&A]
i try to run:
cat cidr.txt  | httpx -ports http:80 -o 80.txt -stats , but i dont know what option to add to save only the open ip.

also i want integrate to scan with a nuclei CVE template.

any help , i will appreciate .
thank you

**Comment by olearycrew** [comment]
>  save only the open ip.

Not sure what you mean here, but httpx will only return those who respond on the specified port. If the host doesn't respond, it won't be in the output.

You can then also pipe the output into nuclei if you like

`nuclei -l 80.txt -id CVE-XX-XXX`

Hope that helps
